config/docker-val.config.src

```diff
@@ -40,6 +40,7 @@
   ]},
  {miner,
   [
+   {jsonrpc_ip, {0,0,0,0}}, %% bind to host for docker containers
```

Comment:
nice, thank you! Could do do this for the `docker-testval` and `docker` configs too?

Reply:
Done. Thanks for pointing that out.